crates/aranya-client/src/aqc/net.rs

```
@@ -339,7 +340,7 @@ impl AqcClient {
                     .send(Bytes::from(ack_bytes))
                     .await
                     .map_err(AqcError::from)?;
-                stream.close().await.map_err(AqcError::from)?;
+                let _ = stream.close().await;
```

Lets maybe warn on this, I dont like eating the error completely.